src/Akka.Persistence.Azure/Journal/AzureTableStorageJournal.cs

```diff
@@ -70,16 +71,16 @@ public AzureTableStorageJournal(Config config = null)
             if (multiSetup.HasValue)
             {
                 var journalId = Self.Path.Name.SplitDottedPathHonouringQuotes().Last();
-                setup = multiSetup.Value.Get(journalId);
+                setup = Option<AzureTableStorageJournalSetup>.Create(multiSetup.Value!.Get(journalId)!);
                 if(setup.HasValue)
                     _settings = setup.Value.Apply(_settings);
             }
             
             _serialization = new SerializationHelper(Context.System);
 
-            if (_settings.Development)
+            if (_settings.TableServiceClientFactory != null)
             {
-                _tableServiceClient = new TableServiceClient(connectionString: "UseDevelopmentStorage=true");
```

Comment:
Dropping the `Development` connection auto-config since development connection string can vary wildly in Azurite, compared to the old Azure Storage Emulator.